donkeycar/parts/gps.py

```diff
@@ -641,8 +678,8 @@ def read_gps():
                     print(".", end="")
```

Comment:
This is obviously difficult in logging....

Reply:
This is in a command line utility, so it is UI, not logging.

Reply:
You're right.